src/useStorage/useStorage.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-redeclare */
```

Comment:
we should really kill that eslint rule

Reply:
I think its a conflicting js vs ts rule. We really need to rework that config...